python/instrumentation/openinference-instrumentation-langchain/src/openinference/instrumentation/langchain/__init__.py

```diff
@@ -64,6 +64,26 @@ def _uninstrument(self, **kwargs: Any) -> None:
     def get_span(self, run_id: UUID) -> Optional[Span]:
         return self._tracer.get_span(run_id) if self._tracer else None
 
+    def get_ancestors(self, run_id: UUID) -> Optional[List[Span]]:
```

Comment:
```suggestion
    def get_ancestor_spans(self, run_id: UUID) -> Optional[List[Span]]:
```
just to be explicit about the return type